src/main.rs

```diff
@@ -530,36 +529,43 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .into());
     }
 
-    let config_directory = solana_root.get_root_path().join("config-k8s");
-    let mut genesis = Genesis::new(config_directory.clone(), genesis_flags);
+    let no_bootstrap = matches.is_present("no_bootstrap");
```

Comment:
nit: it can simplify reading later to call this parameter something like `deploy_bootstrap_validator` and set it to `!matches.is_present("no_bootstrap")`. Double-negatives can cause confusion, ie. `if !no_bootstrap`